impeller/entity/entity_unittests.cc

```diff
@@ -2026,5 +2030,49 @@ TEST_P(EntityTest, SdfText) {
   ASSERT_TRUE(OpenPlaygroundHere(callback));
 }
 
+TEST_P(EntityTest, RuntimeEffect) {
+  if (GetParam() != PlaygroundBackend::kMetal) {
+    GTEST_SKIP_("This test only has a Metal fixture at the moment.");
+  }
+
+  auto callback = [&](ContentContext& context, RenderPass& pass) -> bool {
+    auto contents = std::make_shared<RuntimeEffectContents>();
+    contents->SetGeometry(Geometry::MakeCover());
+
+    // layout(location = 0) uniform float iTime;
+    // layout(location = 1) uniform vec2 iResolution;
+    //
+    // layout(location = 0) out vec4 fragColor;
+    //
+    // void main() {
+    //   // Normalized pixel coordinates (from 0 to 1)
+    //   vec2 uv = gl_FragCoord.xy/iResolution;
+    //   float t = 4 * iTime;
+    //   vec3 col = 0.5 + 0.5*cos(t + uv.xyx + vec3(0,1,4));
+    //   fragColor = vec4(col,1.0);
+    // }
+    auto runtime_stage = LoadFixtureRuntimeStage("example.frag.metal.iplr");
```

Comment:
This doesn't need to be checked in. It can be generated by the build. We do that for ink_sparkle [here](https://github.com/flutter/engine/blob/main/impeller/fixtures/BUILD.gn#L36).

Reply:
Good call, done.